src/Craned/Craned.cpp

```diff
@@ -441,6 +441,7 @@ void StartServer() {
 
   g_server = std::make_unique<Craned::CranedServer>(g_config.ListenConf);
 
+  g_ctld_client->start_connecting();
```

Comment:
Camel Case. // Once CranedServer has started, Inform Ctld to connect to this craned.